docker/push.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash
 #
 # Build image from the currently checked out version of CritiqueBrainz
-# and push it to the Docker Hub, with an optional tag (by default "latest").
+# and push it to the Docker Hub, with an optional tag (by default "beta").
 #
 # Usage:
-#   $ ./push.sh [tag]
+#   $ ./push.sh [env] [tag]
 
 cd "$(dirname "${BASH_SOURCE[0]}")/../"
 
-TAG_PART=${1:-latest}
-docker build -t metabrainz/critiquebrainz:$TAG_PART --build-arg GIT_COMMIT_SHA=$(git rev-parse HEAD) .
-docker push metabrainz/critiquebrainz:$TAG_PART
+ENV=${1:-beta}
+TAG=${1:-beta}
```

Comment:
You'll want to make TAG look at a second argument in order to give containers version tags. See LB.